src/main/java/it/hospital/project/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package it.hospital.project.controller;
+
+import it.hospital.project.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin
+@RequestMapping("/api/users")
```

Comment:
We structure application in domained way. So it means, you don't divide it by catalogues "services", "repositories", "controllers", but rather as domains, so "appusers" (where you put everything that is related to users, f.e models,services, controllers and so on), "medicalrecords". The structure you proposed becomes unefficient as project grows.

Reply:
Can i create catalog "appusers" and then put there other catalogues such as "services" "repositories" or should I upload only file, classes?